toc.md

```diff
@@ -1,26 +1,34 @@
-# How to contribute to open source as a corporation
+# Outbound Open Source: Leveraging open source ecosystems
 
-* [Introduction](Introduction.md)
-  * Goal and target audience of the document (managing open source work in corporations)
-  * Why to contribute?
-  * Maturity levels (archtetypes?)
-* [Contribution process](Contributions-to-existing-projects.md)
-  * Responsibility (decision rests with unit)
-  * General structure and scope of the process (Lean Procedure and boundary conditions)
-  * Process for expressing the company approval for contributions
-    * Why is it needed?
-    * Outbound CLA
-    * Procedure for contributions to existing projects
-  * Contribution models
-    * Small contributions Model or trivial contributions
-    * Major to major release model
-    * Full trust model
-    * Spare time contributions
-  * Trainings
-* [Starting new open source project](starting-oss-projects.md)
-  * Why to start own projects? Overview of different variants (attract community, transparency, foundation-based)
-  * Life cycle
-  * Legal and governance considerations
+---
+
+## [Introduction](Introduction.md)
+
+* Getting involved with open source projects
+   * Defining inbound open source (consuming)
+   * Defining outbound open source (contributing and releasing)
```

Comment:
It's a good idea to define the terms at the beginning. Makes sense to me. So we would need to write two new paragraphs about that.

Reply:
I can add this line at the `introduction.md` file:

"When building an OSPO, organizations need to determine how companies are going to get involved with open source projects. This can come in many shapes, from consuming open source projects (inbound open source), to contributing to existing ones, or even releasing their own (outbound open source). The last two activities imply a higher level of commitment and a deep understanding of how the open-source ecosystem works.

Reply:
Regarding mentioning the term OSPO: So far we haven't used it in the document yet, but described outbound OSS from a general organizational perspective, not implying by what exact organizational form it would be implemented. If we use the term OSPO, I think we need to at least briefly explain it.